src/sentry/static/sentry/app/views/organizationEventsV2/eventDetails.jsx

```diff
@@ -17,23 +17,10 @@ import {getQuery} from './utils';
 const slugValidator = function(props, propName, componentName) {
   const value = props[propName];
   // Accept slugs that look like:
-  // * project-slug:123:latest
-  // * project-slug:123:oldest
-  // * project-slug:123:deadbeef
+  // * project-slug:deadbeef:latest
+  // * project-slug:deadbeef:oldest
   // * project-slug:deadbeef
-  if (value && !/^(?:[^:]+:)?(?:[^:]+):(?:[a-f0-9]+|latest|oldest)$/.test(value)) {
-    return new Error(`Invalid value for ${propName} provided to ${componentName}.`);
-  }
-  return null;
-};
-
-const transactionValidator = function(props, propName, componentName) {
-  const value = props[propName];
-  // Accept slugs that look like:
-  // * project-slug:/some/pathname:latest
-  // * project-slug:a_bare_string:oldest
-  // * project-slug:/some/otherpath:deadbeef
-  if (value && !/^(?:[^:]+):(?:[^:]+):(?:[^:]+|latest|oldest)$/.test(value)) {
+  if (value && !/^(?:[^:]+):(?:[a-f0-9]+)(?:\:latest|oldest)?$/.test(value)) {
```

Comment:
Is the project slug part of that right? Can we have `:` in project slug? Is every other character there really valid? If you wanted to, could also enforce that event ID has a length of 32

Reply:
Project slugs can't have `:` in them. The slug generation done in project.save() removes the `:` from the slug.

Reply:
@markstory to double check, are the colons are removed on the django side?

Reply:
There is still one `:` left between the projectid and eventid. We need to retain that one as eventids are only unique within a project.